sycl/unittests/pi/cuda/test_sampler_properties.cpp

```diff
@@ -0,0 +1,127 @@
+//==---- PlatformTest.cpp --- PI unit tests --------------------------------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#define GTEST_HAS_COMBINE 1
```

Comment:
I guess we need this line due to a bit outdated Google Test in LLVM monorepo. But can we possibly use CMake's `target_compile_definitions` for that?

Reply:
Sounds like a good idea to me. I'll take a look.

Reply:
Done